test/jestEnv.js

```diff
@@ -16,6 +16,7 @@ SOLUTION_VERSION='v1.0.0_dev'
 
 // controlpalne backend API
 process.env.AWS_REGION = 'us-east-1'
+process.env.AWS_ACCOUNT_ID = '555555555555'
```

Comment:
add it to allowed list

https://github.com/awslabs/clickstream-analytics-on-aws/blob/main/.gitallowed